src/components/Nav/Nav.js

```diff
@@ -99,8 +112,8 @@ const SearchAndLogin = styled.div`
     font-weight: 700;
     color: ${({ theme }) => theme.colors.black};
     ${({ theme }) => theme.desktop`
-      display:inline;
-      `};
+          display:inline;
+          `};
```

Comment:
```suggestion
     display:inline;
   `};
```
indent가 자동으로 변한거 같은데 확인 부탁드려요